### PR TITLE
TW-980 IA reorg for collections

### PR DESCRIPTION
**Pull request by jkonrath-postman**

### CODE REVIEW

File: redirects.json [hunk 1]
```diff
@@ -2062,5 +2062,73 @@
...
+		"to": "/docs/collections/running-collections/collection-webhooks/"
+	},
+	{
+		"from": "/docs/running-collections/intro-to-collection-runs/",
+		"to": "/docs/collections/running-collections/intro-to-collection-runs/"
+	},
+	{
+		"from": "/docs/collections/running-collections/running-collections-overview/",
+		"to": "/docs/collections/running-collections/running-collections-overview/"
+	},
```
Comment (MatthewPinsker-postman) [inline redirects.json:2081]: This redirects back to the same page.

File: redirects.json [hunk 2]
```diff
@@ -1329,51 +1329,51 @@
 	},
 	{
 		"from": "/docs/postman/team-library/activity-feed-and-restoring-collections/",
-		"to": "https://github.com/postmanlabs/postman-docs/blob/9ae76e2efe0a31e08646d5e2e96a42fe67e14573/src/pages/docs/postman/team-library/activity-feed-and-restoring-collections.md"
+		"to": "https://github.com/postmanlabs/postman-docs/blob/9ae76e2efe0a31e08646d5e2e96a42fe67e14573/"
```
Comment (MatthewPinsker-postman) [inline redirects.json:1332]: This URL returns a 404 error message in GitHub. It looks like we mention the same URL in multiple places in this file.

File: src/pages/docs/collections/collections-overview.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+---
+
+_Postman Collections_ are a group of saved requests. Every request you send in Postman appears under the **History** tab of the sidebar. On a small scale, reusing requests through the history section is convenient. As your Postman usage grows, it can be time-consuming to find a particular request in your history. Instead of scrolling through your history section, you can save all your requests as a group for easier access.
+
+## Contents
+
+* [Creating your first collection](#creating-your-first-collection)
+* [Using collection](#using-collections)
+* [Running collections](#running-collections)
+* [Running collections on the command line with Newman](#running-collections-on-the-command-line-with-newman)
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/collections/collections-overview.md:26]: Did we intend to add this as an H2 towards the bottom?

File: src/pages/docs/getting-started/creating-the-first-collection.md [hunk 4]
```diff
@@ -69,3 +69,9 @@ After you save the request, your new collection and the request are listed under
 > You can also create a new collection from your workspace's **Overview** tab. Under **In this workspace**, hover over **Collections** and select the **+** icon.
 >
 > <img alt="Create new request overview" src="https://assets.postman.com/postman-docs/create-new-collection-v9.jpg" width="300px"/>
+
+## Next steps
+
+* To learn more about collections, visit the [Collections overview](/docs/collections/collections-overview/).
+* To create, edit, delete, and work with your collections, visit [Using collections](/docs/collections/using-collections.md).
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/getting-started/creating-the-first-collection.md:76]:
```suggestion
* To create, edit, delete, and work with your collections, visit [Using collections](/docs/collections/using-collections/).
```

File: src/pages/docs/collections/running-collections/intro-to-collection-runs.md [hunk 5]
```diff
@@ -167,15 +167,15 @@ To export a collection run, do the following:
...
+In addition to running collections manually, the Collection Runner enables you to [schedule collections](/docs/collections/running-collections/scheduling-collection-runs/) to run automatically at specified times in the Postman cloud. You can also use collection runs in conjunction with other Postman utilities to build automation into your API projects.
 
 * [The Postman CLI](/docs/postman-cli/postman-cli-overview/) command-line interface enables you to run collections and build them into your development pipeline, responding to test outcomes to support your API performance.
-* The [Newman](/docs/running-collections/using-newman-cli/command-line-integration-with-newman/) interface also enables you to run collections from the command line.
+* The [Newman](/docs/collections//using-newman-cli/command-line-integration-with-newman/) interface also enables you to run collections from the command line.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/collections/running-collections/intro-to-collection-runs.md:173]:
```suggestion
* The [Newman](/docs/collections/using-newman-cli/command-line-integration-with-newman/) interface also enables you to run collections from the command line.
```

File: src/pages/docs/collections/running-collections/running-collections-overview.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    url: "https://youtu.be/tDQzY1Hn2LY"
+  - type: dynamic_blog
+    name: "Blog posts"
+    blog_tag: "monitors"
+
+warning: false
+
+---
+
+In Postman, a [collection](/docs/getting-started/creating-the-first-collection/) can be a group of saved [requests](/docs/getting-started/sending-the-first-request/), or a [workflow](/docs/collections/running-collections/building-workflows/), or a [test suite](/docs/writing-scripts/test-scripts/). You might run a collection to send requests to your new API's endpoints, or to retrieve data from a data source, or to test an API's functionality. Postman enables you to run collections in several ways:
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/collections/running-collections/running-collections-overview.md:25]:
```suggestion
In Postman, a [collection](/docs/getting-started/creating-the-first-collection/) can be a group of saved [requests](/docs/getting-started/sending-the-first-request/), a [workflow](/docs/collections/running-collections/building-workflows/), or a [test suite](/docs/writing-scripts/test-scripts/). You might run a collection to send requests to your new API's endpoints, to retrieve data from a data source, or to test an API's functionality. Postman enables you to run collections in several ways:
```

File: src/pages/docs/collections/running-collections/running-collections-overview.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+* You can [run collections manually](/docs/collections/running-collections/intro-to-collection-runs/), with the Collection Runner. This is useful when you're sending a small number of requests and you don't need to repeat them many times.
+* You can run collections frequently in an automated way by scheduling collection runs or by setting up monitors. [Scheduled collection runs](/docs/collections/running-collections/scheduling-collection-runs) are useful for automating testing, and [monitors](/docs/collections/running-collections/scheduling-collection-runs-monitors) are useful for checking performance, availability, and reliability.
+* You can automate collection runs in your CI/CD with the [Postman CLI](/docs/postman-cli/postman-cli-overview/) or [Newman](/docs/collections//using-newman-cli/command-line-integration-with-newman/).
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/collections/running-collections/running-collections-overview.md:29]:
```suggestion
* You can automate collection runs in your CI/CD with the [Postman CLI](/docs/postman-cli/postman-cli-overview/) or [Newman](/docs/collections/using-newman-cli/command-line-integration-with-newman/).
```

File: src/pages/docs/collections/using-newman-cli/command-line-integration-with-newman.md [hunk 8]
```diff
@@ -57,16 +57,16 @@ You can also run collections from the command line using the [Postman CLI](/docs
 
 ## Getting started
 
-To get started, install Node.js and Newman, then run your collections. Learn more about [installing and running Newman](/docs/running-collections/using-newman-cli/installing-running-newman/).
+To get started, install Node.js and Newman, then run your collections. Learn more about [installing and running Newman](/docs/collections//using-newman-cli/installing-running-newman/).
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/collections/using-newman-cli/command-line-integration-with-newman.md:60]:
```suggestion
To get started, install Node.js and Newman, then run your collections. Learn more about [installing and running Newman](/docs/collections/using-newman-cli/installing-running-newman/).
```

File: src/pages/docs/collections/using-newman-cli/command-line-integration-with-newman.md [hunk 9]
```diff
@@ -57,16 +57,16 @@ You can also run collections from the command line using the [Postman CLI](/docs
...
 
 ## Options
 
-Newman provides a rich set of options to customize a run. Learn more about [Newman options](/docs/running-collections/using-newman-cli/newman-options/).
+Newman provides a rich set of options to customize a run. Learn more about [Newman options](/docs/collections//using-newman-cli/newman-options/).
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/collections/using-newman-cli/command-line-integration-with-newman.md:64]:
```suggestion
Newman provides a rich set of options to customize a run. Learn more about [Newman options](/docs/collections/using-newman-cli/newman-options/).
```

File: src/pages/docs/collections/using-newman-cli/command-line-integration-with-newman.md [hunk 10]
```diff
@@ -57,16 +57,16 @@ You can also run collections from the command line using the [Postman CLI](/docs
...
 
 ## File uploads
 
-Newman supports file uploads, so you can use a data file (such as a text file) to fill in form data fields. Learn more about [uploading files in Newman](/docs/running-collections/using-newman-cli/newman-file-uploads/).
+Newman supports file uploads, so you can use a data file (such as a text file) to fill in form data fields. Learn more about [uploading files in Newman](/docs/collections//using-newman-cli/newman-file-uploads/).
 
 ## Custom reporters
 
-Custom reporters are useful to generate collection run reports that cater to specific use cases, for example, logging out the response body when a request (or its tests) fail. Learn more about [using Newman custom reporters](/docs/running-collections/using-newman-cli/newman-custom-reporters/).
+Custom reporters are useful to generate collection run reports that cater to specific use cases, for example, logging out the response body when a request (or its tests) fail. Learn more about [using Newman custom reporters](/docs/collections//using-newman-cli/newman-custom-reporters/).
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/collections/using-newman-cli/command-line-integration-with-newman.md:72]:
```suggestion
Custom reporters are useful to generate collection run reports that cater to specific use cases, for example, logging out the response body when a request (or its tests) fail. Learn more about [using Newman custom reporters](/docs/collections/using-newman-cli/newman-custom-reporters/).
```

File: src/pages/docs/collections/using-newman-cli/continuous-integration.md [hunk 11]
```diff
@@ -9,7 +9,7 @@ contextual_links:
     url: "/docs/sending-requests/intro-to-collections/"
   - type: link
     name: "Command-line integration with Newman"
-    url: "/docs/running-collections/using-newman-cli/command-line-integration-with-newman/"
+    url: "/docs/collections//using-newman-cli/command-line-integration-with-newman/"
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/collections/using-newman-cli/continuous-integration.md:12]:
```suggestion
    url: "/docs/collections/using-newman-cli/command-line-integration-with-newman/"
```

File: src/pages/docs/collections/using-newman-cli/continuous-integration.md [hunk 12]
```diff
@@ -54,7 +54,7 @@ Follow the [steps to download Node](https://nodejs.org/en/download/package-manag
 
 ## Step 2: Install Newman
 
-[Newman](/docs/running-collections/using-newman-cli/command-line-integration-with-newman/) is a command-line tool that allows you to run a collection in your local development environment or on your own server. The following command installs Newman in your CI.
+[Newman](/docs/collections//using-newman-cli/command-line-integration-with-newman/) is a command-line tool that allows you to run a collection in your local development environment or on your own server. The following command installs Newman in your CI.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/collections/using-newman-cli/continuous-integration.md:57]:
```suggestion
[Newman](/docs/collections/using-newman-cli/command-line-integration-with-newman/) is a command-line tool that allows you to run a collection in your local development environment or on your own server. The following command installs Newman in your CI.
```

File: src/pages/docs/collections/using-newman-cli/installing-running-newman.md [hunk 13]
```diff
@@ -107,7 +107,7 @@ $ newman run mycollection.json --reporters cli,json --reporter-json-export outpu
 
 ## Options
 
-Newman provides a rich set of options to customize a run. Learn more at [Newman options](/docs/running-collections/using-newman-cli/newman-options/)
+Newman provides a rich set of options to customize a run. Learn more at [Newman options](/docs/collections//using-newman-cli/newman-options/)
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/collections/using-newman-cli/installing-running-newman.md:110]:
```suggestion
Newman provides a rich set of options to customize a run. Learn more at [Newman options](/docs/collections/using-newman-cli/newman-options/)
```

File: src/pages/docs/collections/using-newman-cli/integration-with-jenkins.md [hunk 14]
```diff
@@ -35,7 +35,7 @@ tags:
 updated: 2022-01-14
 ---
 
-Postman contains a full-featured [testing sandbox](/docs/writing-scripts/script-references/postman-sandbox-api-reference/) that enables you to write and execute JavaScript based tests for your API. You can then integrate Postman with your CI/CD build system using [Newman](/docs/running-collections/using-newman-cli/command-line-integration-with-newman/), the command-line collection runner for Postman.
+Postman contains a full-featured [testing sandbox](/docs/writing-scripts/script-references/postman-sandbox-api-reference/) that enables you to write and execute JavaScript based tests for your API. You can then integrate Postman with your CI/CD build system using [Newman](/docs/collections//using-newman-cli/command-line-integration-with-newman/), the command-line collection runner for Postman.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/collections/using-newman-cli/integration-with-jenkins.md:38]:
```suggestion
Postman contains a full-featured [testing sandbox](/docs/writing-scripts/script-references/postman-sandbox-api-reference/) that enables you to write and execute JavaScript based tests for your API. You can then integrate Postman with your CI/CD build system using [Newman](/docs/collections/using-newman-cli/command-line-integration-with-newman/), the command-line collection runner for Postman.
```

File: src/pages/docs/collections/using-newman-cli/integration-with-travis.md [hunk 15]
```diff
@@ -8,7 +8,7 @@ contextual_links:
     name: "Prerequisites"
   - type: link
     name: "Running collections on the command line with Newman"
-    url: "/docs/running-collections/using-newman-cli/command-line-integration-with-newman/"
+    url: "/docs/collections//using-newman-cli/command-line-integration-with-newman/"
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/collections/using-newman-cli/integration-with-travis.md:11]:
```suggestion
    url: "/docs/collections/using-newman-cli/command-line-integration-with-newman/"
```

File: src/pages/docs/collections/using-newman-cli/integration-with-travis.md [hunk 16]
```diff
@@ -39,7 +39,7 @@ With CI, every code commit triggers an automated process that builds the code an
 
 ## Running Postman tests with Travis CI
 
-In general, integrating your [Postman tests](/docs/writing-scripts/test-scripts/) with Travis CI is the same process as [running on Jenkins](/docs/running-collections/using-newman-cli/integration-with-jenkins/), AppVeyor, or any other build system. You will set up your CI configuration to run a shell command when starting your build. The command is a [Newman script](/docs/running-collections/using-newman-cli/command-line-integration-with-newman/) that runs the collection containing the tests and returns a pass or fail exit code that’s logged in your CI system.
+In general, integrating your [Postman tests](/docs/writing-scripts/test-scripts/) with Travis CI is the same process as [running on Jenkins](/docs/collections//using-newman-cli/integration-with-jenkins/), AppVeyor, or any other build system. You will set up your CI configuration to run a shell command when starting your build. The command is a [Newman script](/docs/collections//using-newman-cli/command-line-integration-with-newman/) that runs the collection containing the tests and returns a pass or fail exit code that’s logged in your CI system.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/collections/using-newman-cli/integration-with-travis.md:42]:
```suggestion
In general, integrating your [Postman tests](/docs/writing-scripts/test-scripts/) with Travis CI is the same process as [running on Jenkins](/docs/collections/using-newman-cli/integration-with-jenkins/), AppVeyor, or any other build system. You will set up your CI configuration to run a shell command when starting your build. The command is a [Newman script](/docs/collections/using-newman-cli/command-line-integration-with-newman/) that runs the collection containing the tests and returns a pass or fail exit code that’s logged in your CI system.
```

File: src/pages/docs/collections/using-newman-cli/newman-with-docker.md [hunk 17]
```diff
@@ -8,7 +8,7 @@ contextual_links:
     name: "Prerequisites"
   - type: link
     name: "Command-line integration with Newman"
-    url: "/docs/running-collections/using-newman-cli/command-line-integration-with-newman/"
+    url: "/docs/collections//using-newman-cli/command-line-integration-with-newman/"
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/collections/using-newman-cli/newman-with-docker.md:11]:
```suggestion
    url: "/docs/collections/using-newman-cli/command-line-integration-with-newman/"
```

File: src/pages/docs/collections/using-newman-cli/newman-with-docker.md [hunk 18]
```diff
@@ -78,8 +78,8 @@ Check out [how to run Newman in Docker for Windows](https://blog.postman.com/usi
...
 
-* [Using the Collection Runner](/docs/running-collections/intro-to-collection-runs/)
-* [Working with data files](/docs/running-collections/working-with-data-files/)
-* [Building workflows](/docs/running-collections/building-workflows/)
-* [Integration with Jenkins](/docs/running-collections/using-newman-cli/integration-with-jenkins/)
-* [Integration with Travis CI](/docs/running-collections/using-newman-cli/integration-with-travis/)
+* [Using the Collection Runner](/docs/collections/running-collections/intro-to-collection-runs/)
+* [Working with data files](/docs/collections/running-collections/working-with-data-files/)
+* [Building workflows](/docs/collections/running-collections/building-workflows/)
+* [Integration with Jenkins](/docs/collections//using-newman-cli/integration-with-jenkins/)
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/collections/using-newman-cli/newman-with-docker.md:84]:
```suggestion
* [Integration with Jenkins](/docs/collections/using-newman-cli/integration-with-jenkins/)
```

File: src/pages/docs/collections/using-newman-cli/newman-with-docker.md [hunk 19]
```diff
@@ -78,8 +78,8 @@ Check out [how to run Newman in Docker for Windows](https://blog.postman.com/usi
...
+* [Integration with Travis CI](/docs/collections//using-newman-cli/integration-with-travis/)
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/collections/using-newman-cli/newman-with-docker.md:85]:
```suggestion
* [Integration with Travis CI](/docs/collections/using-newman-cli/integration-with-travis/)
```

File: src/pages/docs/developer/collection-conversion.md [hunk 20]
```diff
@@ -21,13 +21,13 @@ contextual_links:
     name: "Next steps"
   - type: link
     name: "Continuous Integration"
-    url: "/docs/running-collections/using-newman-cli/continuous-integration/"
+    url: "/docs/collections//using-newman-cli/continuous-integration/"
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/developer/collection-conversion.md:24]:
```suggestion
    url: "/docs/collections/using-newman-cli/continuous-integration/"
```

File: src/pages/docs/developer/collection-conversion.md [hunk 21]
```diff
@@ -21,13 +21,13 @@ contextual_links:
...
 
 warning: false
 
 ---
 
-You can convert from a variety of API formats into Postman Collections. If you have API requests defined as OpenAPI, Swagger, RAML, or GraphQL schema formats, you can convert these to Postman Collections. You can import Postman Collections into the app or use them with other Postman utilities including [Newman](/docs/running-collections/using-newman-cli/command-line-integration-with-newman/), Postman's command-line collection runner.
+You can convert from a variety of API formats into Postman Collections. If you have API requests defined as OpenAPI, Swagger, RAML, or GraphQL schema formats, you can convert these to Postman Collections. You can import Postman Collections into the app or use them with other Postman utilities including [Newman](/docs/collections//using-newman-cli/command-line-integration-with-newman/), Postman's command-line collection runner.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/developer/collection-conversion.md:30]:
```suggestion
You can convert from a variety of API formats into Postman Collections. If you have API requests defined as OpenAPI, Swagger, RAML, or GraphQL schema formats, you can convert these to Postman Collections. You can import Postman Collections into the app or use them with other Postman utilities including [Newman](/docs/collections/using-newman-cli/command-line-integration-with-newman/), Postman's command-line collection runner.
```

File: src/pages/docs/developer/collection-conversion.md [hunk 22]
```diff
@@ -36,7 +36,7 @@ You can convert from a variety of API formats into Postman Collections. If you h
 
 ## Converting API formats into Postman Collections
 
-Check out the repo for the format you need to convert—the READMEs indicate how to install and use the converters. Your output collections will be suitable for import into Postman, or to run collections on the command line using [Newman](/docs/running-collections/using-newman-cli/command-line-integration-with-newman/), Postman's command-line collection runner.
+Check out the repo for the format you need to convert—the READMEs indicate how to install and use the converters. Your output collections will be suitable for import into Postman, or to run collections on the command line using [Newman](/docs/collections//using-newman-cli/command-line-integration-with-newman/), Postman's command-line collection runner.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/developer/collection-conversion.md:39]:
```suggestion
Check out the repo for the format you need to convert—the READMEs indicate how to install and use the converters. Your output collections will be suitable for import into Postman, or to run collections on the command line using [Newman](/docs/collections/using-newman-cli/command-line-integration-with-newman/), Postman's command-line collection runner.
```

File: src/pages/docs/developer/collection-sdk.md [hunk 23]
```diff
@@ -13,15 +13,15 @@ contextual_links:
     name: "Next steps"
   - type: link
     name: "Running collections on the command line"
-    url: "/docs/running-collections/using-newman-cli/command-line-integration-with-newman/"
+    url: "/docs/collections//using-newman-cli/command-line-integration-with-newman/"
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/developer/collection-sdk.md:16]:
```suggestion
    url: "/docs/collections/using-newman-cli/command-line-integration-with-newman/"
```

File: src/pages/docs/developer/collection-sdk.md [hunk 24]
```diff
@@ -53,4 +53,4 @@ Get started with the Collection SDK from the [project repo](https://github.com/p
 
 If you're working with the Collection SDK to automate part of your collection run workflow, you can integrate your Collection SDK development with Newman, Postman's command-line collection runner:
 
-* To learn more about integrating your Collection SDK development with Newman, visit [Running collections on the command line with Newman](/docs/running-collections/using-newman-cli/command-line-integration-with-newman/).
+* To learn more about integrating your Collection SDK development with Newman, visit [Running collections on the command line with Newman](/docs/collections//using-newman-cli/command-line-integration-with-newman/).
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/developer/collection-sdk.md:56]:
```suggestion
* To learn more about integrating your Collection SDK development with Newman, visit [Running collections on the command line with Newman](/docs/collections/using-newman-cli/command-line-integration-with-newman/).
```

File: src/pages/docs/developer/intro-api.md [hunk 25]
```diff
@@ -43,15 +43,15 @@ contextual_links:
     name: "Next steps"
   - type: link
     name: "Continuous Integration"
-    url: "/docs/running-collections/using-newman-cli/continuous-integration/"
+    url: "/docs/collections//using-newman-cli/continuous-integration/"
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/developer/intro-api.md:46]:
```suggestion
    url: "/docs/collections/using-newman-cli/continuous-integration/"
```

File: src/pages/docs/developer/intro-api.md [hunk 26]
```diff
@@ -43,15 +43,15 @@ contextual_links:
...
 
 warning: false
 
 ---
 
 The [Postman API](https://www.postman.com/postman/workspace/postman-public-workspace/documentation/12959542-c8142d51-e97c-46b6-bd77-52bb66712c9a) endpoints enable you to integrate Postman within your development toolchain.
 
-You can add new collections, update existing collections, update environments, and add and run monitors directly through the API. This enables you to programmatically access data stored in your Postman account. You can also [combine the Postman API with Newman](/docs/running-collections/using-newman-cli/continuous-integration/) to integrate Postman with your CI/CD workflow.
+You can add new collections, update existing collections, update environments, and add and run monitors directly through the API. This enables you to programmatically access data stored in your Postman account. You can also [combine the Postman API with Newman](/docs/collections//using-newman-cli/continuous-integration/) to integrate Postman with your CI/CD workflow.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/developer/intro-api.md:54]:
```suggestion
You can add new collections, update existing collections, update environments, and add and run monitors directly through the API. This enables you to programmatically access data stored in your Postman account. You can also [combine the Postman API with Newman](/docs/collections/using-newman-cli/continuous-integration/) to integrate Postman with your CI/CD workflow.
```

File: src/pages/docs/developer/resources-intro.md [hunk 27]
```diff
@@ -44,4 +44,4 @@ Check out the pages in this section for an intro to developer resources that all
 * [**Postman Collection converters**](/docs/developer/collection-conversion/)
     * The converters allow you to transform OpenAPI, Swagger, RAML, and GraphQL specs to Postman Collection format.
 
-> You can use [Newman](/docs/running-collections/using-newman-cli/command-line-integration-with-newman/), Postman's command-line collection runner, in conjunction with other utilities to build collection runs into your development pipeline. Newman is also an [open-source project](https://github.com/postmanlabs/newman) that you can use and contribute to.
+> You can use [Newman](/docs/collections//using-newman-cli/command-line-integration-with-newman/), Postman's command-line collection runner, in conjunction with other utilities to build collection runs into your development pipeline. Newman is also an [open-source project](https://github.com/postmanlabs/newman) that you can use and contribute to.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/developer/resources-intro.md:47]:
```suggestion
> You can use [Newman](/docs/collections/using-newman-cli/command-line-integration-with-newman/), Postman's command-line collection runner, in conjunction with other utilities to build collection runs into your development pipeline. Newman is also an [open-source project](https://github.com/postmanlabs/newman) that you can use and contribute to.
```

File: src/pages/docs/developer/runtime-library.md [hunk 28]
```diff
@@ -13,13 +13,13 @@ contextual_links:
     name: "Next steps"
   - type: link
     name: "Continuous Integration"
-    url: "/docs/running-collections/using-newman-cli/continuous-integration/"
+    url: "/docs/collections//using-newman-cli/continuous-integration/"
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/developer/runtime-library.md:16]:
```suggestion
    url: "/docs/collections/using-newman-cli/continuous-integration/"
```

File: src/pages/docs/developer/runtime-library.md [hunk 29]
```diff
@@ -60,10 +60,10 @@ function (err, run) {
 
 You can use the Runtime library if you need a detailed configuration of your request runs, for example as part of an automation workflow to integrate Postman Collection runs into your development pipeline.
 
-> Note that if you only need to run collections, you can use [Newman](/docs/running-collections/using-newman-cli/command-line-integration-with-newman/), Postman's command-line tool.
+> Note that if you only need to run collections, you can use [Newman](/docs/collections//using-newman-cli/command-line-integration-with-newman/), Postman's command-line tool.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/developer/runtime-library.md:63]:
```suggestion
> Note that if you only need to run collections, you can use [Newman](/docs/collections/using-newman-cli/command-line-integration-with-newman/), Postman's command-line tool.
```

File: src/pages/docs/developer/runtime-library.md [hunk 30]
```diff
@@ -60,10 +60,10 @@ function (err, run) {
...
 
 ## Next steps
 
 Now that you've gotten started with the Runtime library, you might also want to use the Postman API in your continuous integration and continuous deployment (CI/CD) workflow:
 
-* To learn more about incorporating Postman into your CI/CD workflow, visit [CI with Postman API](/docs/running-collections/using-newman-cli/continuous-integration/).
+* To learn more about incorporating Postman into your CI/CD workflow, visit [CI with Postman API](/docs/collections//using-newman-cli/continuous-integration/).
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/developer/runtime-library.md:69]:
```suggestion
* To learn more about incorporating Postman into your CI/CD workflow, visit [CI with Postman API](/docs/collections/using-newman-cli/continuous-integration/).
```

File: redirects.json (hunk 2 above)
Comment (jkonrath-postman) [reply redirects.json]: I just fixed all the github ones

**Review by MatthewPinsker-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/getting-started/navigating-postman.md [hunk 31]
```diff
@@ -108,7 +108,7 @@ The Postman sidebar provides access to the fundamental elements of Postman:
 * [APIs](/docs/designing-and-developing-your-api/creating-an-api/)
 * [Environments](/docs/sending-requests/managing-environments/#creating-environments)
 * [Mock servers](/docs/designing-and-developing-your-api/mocking-data/setting-up-mock/)
-* [Monitors](/docs/running-collections/scheduling-collection-runs/)
+* [Monitors](/docs/collections/running-collections/scheduling-collection-runs/)
```
Comment (mark-anthony-postman) [inline src/pages/docs/getting-started/navigating-postman.md:111]:
```suggestion
* [Monitors](/docs/monitoring-your-api/intro-monitors/)
```

File: src/pages/docs/sending-requests/variables.md [hunk 32]
```diff
@@ -109,7 +109,7 @@ In order from broadest to narrowest, these scopes are: _global_, _collection_, _
...
 * **Environment variables** enable you to scope your work to different environments, for example local development versus testing or production. One environment can be active at a time. If you have a single environment, using collection variables can be more efficient, but environments enable you to specify [role-based access levels](/docs/sending-requests/managing-environments/#working-with-environments-as-a-team).
-* **Data variables** come from external CSV and JSON files to define data sets you can use when running collections with [Newman](/docs/running-collections/using-newman-cli/) or the [Collection Runner](/docs/running-collections/intro-to-collection-runs/). Data variables have current values, which don't persist beyond request or collection runs.
+* **Data variables** come from external CSV and JSON files to define data sets you can use when running collections with [Newman](/docs/collections/using-newman-cli/) or the [Collection Runner](/docs/collections/running-collections/intro-to-collection-runs/). Data variables have current values, which don't persist beyond request or collection runs.
```
Comment (mark-anthony-postman) [inline src/pages/docs/sending-requests/variables.md:112]:
```suggestion
* **Data variables** come from external CSV and JSON files to define data sets you can use when running collections with [Newman](/docs/collections/using-newman-cli/command-line-integration-with-newman/) or the [Collection Runner](/docs/collections/running-collections/intro-to-collection-runs/). Data variables have current values, which don't persist beyond request or collection runs.
```

**Review by mark-anthony-postman — APPROVED** [review]
The new structure looks great!

File: src/pages/docs/collections/collections-overview.md [hunk 33]
```diff
@@ -0,0 +1,40 @@
+---
+title: "Collections overview"
```
[the remark this answers is not in the document]
Comment (jkonrath-postman) [reply src/pages/docs/collections/collections-overview.md:2]: Other way around, actually.

**Pull request merged**